packages/spatial/src/renderer/WebGLRendererSystem.tsx

```diff
@@ -314,6 +316,15 @@ const execute = () => {
     _scene.background =
       sessionMode === 'immersive-ar' ? null : renderMode === RenderModes.WIREFRAME ? new Color(0xffffff) : background
 
+    if (renderMode === RenderModes.UNLIT) {
```

Comment:
let's move this to it's own system inserted `{ before: WebGLRendererSystem }`, and use ECS queries to disable each light.